motor/providers/aws/provider.go

```diff
@@ -78,6 +78,9 @@ func New(pCfg *providers.Config, opts ...ProviderOption) (*Provider, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "could not load aws configuration")
 	}
+	if cfg.Region == "" {
+		cfg.Region = "us-east-1" // in case the user has no region set, default to us-east-1
```

Comment:
I would suggest we also log a warning here to show the user we're defaulting to some region. Otherwise it might be confusing for them that suddenly we fetch data from AWS w/o them having set a region either via param or via the aws config

Reply:
👍  good point, ill add that in